### PR TITLE
Add url smoke tests to Kolibri

### PR DESCRIPTION
**Pull request by rtibbles**
### Summary
This adds very minimal URL smoke tests to Kolibri (which only work for URLs that do not need supplied arguments).

Fixes one small issue that arose during testing.

----

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/core/content/api.py [hunk 1]
```diff
@@ -436,11 +436,15 @@ def copies_count(self, request, **kwargs):
         """
         Returns the number of node copies for each content id.
         """
-        content_ids = self.request.query_params.get('content_ids', []).split(',')
```
Comment (indirectlylit) [inline kolibri/core/content/api.py:317]: yup, that ain't right

File: kolibri/core/test/test_key_urls.py [hunk 2]
```diff
@@ -32,3 +35,99 @@ def test_redirect_root_to_learn_if_logged_in(self):
...
+        """
+        This is a very liberal test, we are mostly just concerned with making sure
+        that no pages throw errors (500).
+        Adapted from:
+        http://stackoverflow.com/questions/14454001/list-all-suburls-and-check-if-broken-in-python#answer-19162337
+        Test all pattern in root urlconf and included ones.
+        Do GET requests only.
+        A pattern is skipped if any of the conditions applies:
+            - pattern has no name in urlconf
+            - pattern expects any positinal parameters
```
Comment (indirectlylit) [inline kolibri/core/test/test_key_urls.py]: typo

File: kolibri/core/test/test_key_urls.py [hunk 3]
```diff
@@ -32,3 +35,99 @@ def test_redirect_root_to_learn_if_logged_in(self):
...
+                                      "{url} gave status code {status_code}".format(url=url, status_code=response.status_code))
+                        # print status code if it is not 200
+                        status = "" if response.status_code == 200 else str(response.status_code) + " "
+                        print(status + url)
+                        if url == reverse('kolibri:logout'):
+                            self.client.login(**credentials)
+                    except NoReverseMatch:
+                        pass
+                else:
+                    print("SKIP " + regex.pattern + " " + fullname)
```
Comment (indirectlylit) [inline kolibri/core/test/test_key_urls.py]:
```python
print("SKIP", regex.pattern, fullname)
```

**Review by indirectlylit — APPROVED** [review]
looks useful

File: kolibri/core/test/test_key_urls.py (hunk 2 above)
Comment (rtibbles) [reply kolibri/core/test/test_key_urls.py]: Copypasta problems.

**Comment by rtibbles** [comment]
Note: tests are currently failing because there are no webpack assets, so every page with webpack assets is giving a 500.

**Comment by indirectlylit** [comment]
>  tests are currently failing because there are no webpack assets

I'm not totally sure what you mean, and whether this is a temporary condition?

**Pull request merged**